Client/src/App.jsx

```diff
@@ -0,0 +1,44 @@
+import styles from './main.css';
+import { useState, useEffect } from 'react';
+import { BrowserRouter as Router, Switch, Route } from 'react-router-dom';
+import Header from './components/layout/Header/Header';
+import Wrapper from './components/layout/Wrapper/Wrapper';
+import Info from './components/Pages/Info/Info';
+import Feedback from './components/Pages/Feedback/Feedback';
+import Explore from './components/Pages/Explore/Explore';
+import Home from './components/Pages/Home/Home';
+import ThemeToggler from './components/layout/ThemeToggler/ThemeToggler';
+
+function App() {
+  const [theme, setTheme] = useState('dark');
+  const toggleThemeHandler = () => {
+    theme === '' ? setTheme('dark') : setTheme('');
+  };
+
+  return (
+    <Router>
+      <div className={`main ${theme}`}>
```

Comment:
1) I suggest using [].join(' ') approach here, because the moment you start introducing conditions for your classnames  - the code becomes less and less readable
2) what if theme or any other used in that way gets undefined? default value will help  + .join method

Reply:
Good point, thank you.